db/migrate/20221101182844_add_user_to_comment.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] kindly reference author instead of user like this ` t.references :author, foreign_key: { to_table: 'users' }`

Reply:
Hi @VICTOR-CHUKWU , firstly, thanks for your time and review. :smiley:

I have a question:
Is there a way to change the migrations so this would already be set up correctly? Or the only way is to modify the code directly in these parts?

You can see the migrations that I used in a file called ``migrations.txt``

:star: Thanks again!

Reply:
Done!